components/calculator/misdemeanor/QandAContainer.tsx

```diff
@@ -1,26 +1,39 @@
 import {
   Box, Button, Container, Stack,
 } from '@mui/material';
-import { PortableText } from '@portabletext/react';
-import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
You can delete both of the eslint disable lines in this file. They are unneccesary in a weird glitch that I also experience sometimes where the imports don't resolve right away but after restarting the red squigglies go away. When I delete the eslint disable comments, I do not get any eslint warnings.

Reply:
Done